mixins/form.jsx

```diff
@@ -411,7 +417,13 @@ module.exports = {
         submitting: true
       });
       submitProps = form.buildProps(props);
-      this.submit("/api/signup", submitProps, this.signupSuccess, this.signupError);
+      this.submit(url, submitProps, success, error);
     }
+  },
+  signup: function(validate, props) {
+    this.doSignup("/api/signup", validate, props, this.signupSuccess, this.signupError);
```

Comment:
My thought process (and it might not be the greatest idea) was making a thunderbird specific path felt weird as it's not really anything specific to thunderbird, hence the name mailchimp, but I also didn't want the main path we use to become obscured behind a specific. So keeping it signup made sense.

It means most cases we won't need to think about  mailchimp or thunderbird, just signup, and can ignore the other.